core/templates/components/skill-selector/merge-skill-modal.component.html

```diff
@@ -5,51 +5,72 @@ <h3>
   <i class="fa fa-times modal-close-button" (click)="cancel()"></i>
 </div>
 <div class="modal-body">
-  <oppia-skill-selector [sortedSkillSummaries]="skillSummaries"
-                        [selectedSkillId]="selectedSkillId"
-                        [countOfSkillsToPrioritize]="countOfSkillsToPrioritize"
-                        [categorizedSkills]="categorizedSkills"
-                        [untriagedSkillSummaries]="untriagedSkillSummaries"
-                        [allowSkillsFromOtherTopics]="allowSkillsFromOtherTopics"
-                        (selectedSkillIdChange)="setSelectedSkillId($event)">
+  <oppia-skill-selector [sortedSkillSummaries]="skillSummaries" [selectedSkillId]="selectedSkillId"
+    [countOfSkillsToPrioritize]="countOfSkillsToPrioritize" [categorizedSkills]="categorizedSkills"
+    [untriagedSkillSummaries]="untriagedSkillSummaries" [allowSkillsFromOtherTopics]="allowSkillsFromOtherTopics"
+    (selectedSkillIdChange)="setSelectedSkillId($event)">
```

Comment:
Why is this reformatted?

Reply:
I'll roll it back because I made a mistake while fixing lint checks.